docs/advanced-topics/tools-vs-workflows.rst

```diff
@@ -1,53 +1,60 @@
+|                        |  - The Dockerfile for the task's image       |                                               |                                            |
++------------------------+----------------------------------------------+-----------------------------------------------+--------------------------------------------+
+| Nextflow               | N/A                                          | N/A                                           | Any valid Nextflow workflow                |
++------------------------+----------------------------------------------+-----------------------------------------------+--------------------------------------------+
+| Galaxy                 | N/A*                                         | N/A                                           | Any valid Galaxy workflow                  |
++------------------------+----------------------------------------------+-----------------------------------------------+--------------------------------------------+
 
 \* There are tools that make up Galaxy workflows from the Galaxy toolbox or ToolShed.
 Dockstore currently does not support registration of these tools.
 
 
-+------------------------+------------------------------------------+-------------------------------------------------+
-| Dockstore Support      | Tool                                     | Workflow                                        |
-+========================+==========================================+=================================================+
-| "Launch with" support  |  Only via Dockstore CLI                  |  Dockstore CLI & "launch with" buttons on entry |
-+------------------------+------------------------------------------+-------------------------------------------------+
-| Versioning             |  Based off of image's tags               |  Based off of branches/tags from Git repository,|
-|                        |                                          |                                                 |
-|                        |                                          |  even if one of the tasks executes in a Docker  |
-|                        |                                          |  image.                                         |
-+------------------------+------------------------------------------+-------------------------------------------------+
++----------------------+-----------------------------+----------------------------------------+----------------------------------------+
+| Dockstore Support    | Legacy Tool                 | GitHub App Tool                        | Workflow                               |
++======================+=============================+========================================+========================================+
+| Dockstore CLI        |  run in tool mode           |  run in workflow mode                  | run in workflow mode                   |
```

Comment:
I was following the existing pattern that was there when I updated this, but I think the phrases should start with capitals, e.g., "Run in tool mode" vs. "run in tool mode". It would make it consistent with the table above.